tasks/Task.cpp

```diff
@@ -56,9 +55,33 @@ void bms_victron_smart_shunt::Task::cleanupHook()
     TaskBase::cleanupHook();
 }
 
+static BatteryStatus toBatteryStatus(SmartShuntFeedback const& feedback,
+    double max_current)
+{
+    BatteryStatus battery_status;
+    battery_status.time = feedback.timestamp;
+    battery_status.temperature = feedback.temperature;
+    battery_status.charge = feedback.state_of_charge / 100;
+    battery_status.current = feedback.current;
+    battery_status.voltage = feedback.voltage;
+    battery_status.max_current = max_current;
```

Comment:
I dont think that max current is the same thing you are doing here. To me, it seems that max `battery_status.max_current` should come from the sensor itself, but I might be wrong.

Reply:
Isn't he updating the m_max_current according to the feedback current of each loop? It seems like the sensor doesn't send a max_current value and this is a workaround to store this value.

Reply:
Yeah, my point is that the work around is misinformation, its better to not send anything in that field

Reply:
Already talked with sylvain about that. He agrees to hold the max current value in the task

Reply:
Fair enough